pkg/serviceaccount/tokens_controller.go

```diff
@@ -293,6 +293,9 @@ func (e *TokensController) createSecret(serviceAccount *api.ServiceAccount) erro
 		return err
 	}
 	secret.Data[api.ServiceAccountTokenKey] = []byte(token)
+	if e.rootCAProvider != nil {
+		secret.Data[api.ServiceAccountRootCAKey] = e.rootCAProvider()
```

Comment:
should we only set the key if the length of the returned byte slice is greater than 0?